dashboarddoc.md

```diff
@@ -10,14 +10,44 @@ User Dashboard Interaction
 This document illustrates how a Solar Forecast Arbiter user may use the
 Dashboard to complete the following actions:
 
--   [Create a new Site](#create-a-site).
+-   [Create New Site](#create-new-site).
 
--   [Create a new Observation or Forecast](#create-an-observation-or-forecast).
+-   [Create New Observation or Forecast](#create-new-observation-or-forecast).
 
 -   [Upload Data to an Observation or Forecast](#upload-data).
 
+### User Guidance Considerations
+To assist in the user in providing the correct information and
+navigating the dashboard's interface, we've adopted the following
+features not otherwise illustrated in this guide.
 
-Create a Site
+-  To reduce clutter, the dashboard will utilize *progressive
+   disclosure* to hide unnecessary fields from view e.g. the
+   Site form will not display the *PV Modeling Paremeters* fields
+   unless a *Site Type* of *Power Plant* is selected.
+
+-  Descriptive tooltips will be displayed when a user selects a field.
+   <img class="my-3" src="/images/TooltipDetail.png" class="figure"/>
+   <figcaption class="figure pb-2">The latitude field is selected and displays
+   a tooltip describing the data a user should enter.</figcaption>
+
+-  Basic validation will be performed on the entered data to provide
+   immediate feedback to the user. There are two possible sources of
+   error in dashboard interaction:
+
+   1.  Incorrect Input type
+
+       For example, a user entering a non-numeric value into numeric
+       field. In this case, the user would be blocked from submitting
+       their data until a valid value was provided.
+
+   2.  Input does not meet Solar Forecast Arbiter specifications
+
+	   For example, a value greater than 180 is entered into the
+	   longitude field. In this case the user would be notified
+       of the error after submission and must resubmit.
```

Comment:
Maybe you have something in mind here, but I don't have a good idea of how this is going to work. Seems like the form will probably need enough client-side validation that it's highly likely that a submission will be successful. People will not be happy if they have to reenter their data.

Reply:
I think this can be done without users having to reenter their values by redirecting the users back to the form page with their data and the errors received from the API. In which case this would have the same effect as the incorrect input type validation for the user, save for a brief pause while the API request is completed. I think I was trying to differentiate between in-browser validation via the *input* tag's type attribute and the actual validation done by our code, which isn't an important distinction in this document.